modules/prebidmanagerAnalyticsAdapter.js

```diff
@@ -99,6 +99,16 @@ function collectUtmTagData() {
   return pmUtmTags;
 }
 
+function collectPageInfo() {
+  const pageInfo = {
+    domain: window.location.hostname,
+  }
+  if (document.referrer) {
+    pageInfo.referrerDomain = utils.parseUrl(document.referrer).hostname;
+  }
+  return pageInfo;
+}
```

Comment:
would be nice to add a unit test for this.

Reply:
added a test in test/spec/modules/prebidmanagerAnalyticsAdapter_spec.js